git_machete/cli.py

```diff
@@ -615,25 +617,29 @@ def launch(orig_args: List[str]) -> None:
             github_subcommand = parsed_cli.subcommand
             machete_client.read_definition_file()
 
-            if 'draft' in parsed_cli and github_subcommand not in {'create-pr'}:
-                raise MacheteException(
-                    "'--draft' option is only valid with 'create-pr' subcommand.")
-            if 'pr_no' in parsed_cli and github_subcommand not in {'checkout-prs'}:
-                raise MacheteException(
-                    "'pr_no' argument is only valid with 'checkout-prs' subcommand.")
+            if 'draft' in parsed_cli and github_subcommand != 'create-pr':
+                raise MacheteException("'--draft' option is only valid with 'create-pr' subcommand.")
+            for command in ('all', 'by', 'mine'):
+                if command in parsed_cli and github_subcommand != 'checkout-prs':
+                    raise MacheteException(f"'--{command}' argument is only valid with 'checkout-prs' subcommand.")
+            if 'pr_no' in parsed_cli and github_subcommand != 'checkout-prs':
+                raise MacheteException("'pr_no' option is only valid with 'checkout-prs' subcommand.")
 
             if github_subcommand == "anno-prs":
                 machete_client.sync_annotations_to_github_prs()
             elif github_subcommand == "checkout-prs":
-                if 'pr_no' not in parsed_cli:
+                if len(parsed_cli_as_dict) > 3 or len(parsed_cli_as_dict) == 2:
                     raise MacheteException(
-                        "Argument to `git machete github checkout-prs` cannot be"
-                        " empty; expected PR number.")
+                        f"'checkout-prs' subcommand can take only one of following options: {', '.join(['--all', '--by', '--mine', 'pr-no'])}")
                 try:
-                    pr_no: int = int(parsed_cli.pr_no)
+                    if 'pr-no' in parsed_cli:
+                        map(int, parsed_cli.pr_no)
                 except ValueError:
-                    raise MacheteException("PR number is not integer value!")
-                machete_client.checkout_github_prs(pr_no)
+                    raise MacheteException("One of given PR numbers is not integer value!")
```

Comment:
Oh! I'd expect that `git machete github checkout-prs wnfhiufqufg` mentioned above would fail here, but it doesn't

Reply:
Wow with multiple non-numeric PRs the output's different:

```
$ git machete github checkout-prs wnfhiufqufg ewfp3k
Warn: PR #wnfhiufqufg is not found in repository VirtusLab/git-machete, skipping.
Warn: PR #ewfp3k is not found in repository VirtusLab/git-machete, skipping.
Given PRs: wnfhiufqufg, ewfp3k,  are not found in repository VirtusLab/git-machete
```

Pls fix the minor character glitches (like trailing `,` + double space)